bindings/index.template.html

```diff
@@ -3,6 +3,7 @@
 
 <head>
     <meta charset='utf-8'>
+    <meta name="color-scheme" content="light dark" />
```

Comment:
```suggestion
    <meta name="color-scheme" content="light dark">
```

Nit: Just because of consistency I think we should use the `meta` term without the ending `/>` as we have the line above. Personally I use closing tags always but since the old code didn't ...

The same would apply for the quotes though also... `"` vs `'`

Reply:
Thanks, I applied that via 4fd2a55bd613839e27441639a23af86c2784e419 :)

Reply:
I also adjusted it in other places now as well. I guess as soon as we have some kind of linting/auto-formatting step here as well, most of these consistency questions will be resolved for good :)